### PR TITLE
docs(components): add examples and correct property name

### LINKED ISSUE #8: Document React components usage
We provide `<Fill>` and `<Modal>`, but they are not documented. From the source code, user cannot discover `<Modal.Body>` and related components. Let's document that so that it's discoverable.

### PR DESCRIPTION
**Pull request by barmac**
Closes #8

### CODE REVIEW

**Review by pinussilvestrus — COMMENTED** [review]
This fixes https://github.com/camunda/camunda-modeler-plugin-helpers/issues/8, right?

File: components.js [hunk 1]
```diff
@@ -3,15 +3,52 @@ if (!window.components) {
 }
 
 /**
- * Fill component.
+ * Fill component. Set `slot` to "toolbar" to include in the top toolbar.
```
Comment (pinussilvestrus) [inline components.js:6]: do we offer any other slots right now? Or is `toolbar` just an example? Otherwise, it might make sense to state this out

Comment (barmac) [reply components.js:6]: AFAIK it's the only one. 

```suggestion
 * Fill component. Set `slot` to "toolbar" to include in the top toolbar (the only one available now).
```
How about this?

Comment (pinussilvestrus) [reply components.js:6]: As long it's the only one it's okay to have it like the first version

**Review by pinussilvestrus — APPROVED** [review]

**Pull request merged**